app_dart/lib/src/service/config.dart

```diff
@@ -49,6 +49,12 @@ class Config {
         pluginsSlug,
       };
 
+  /// List of guaranteed scheduling Github repos.
+  static Set<gh.RepositorySlug> get guaranteedSchedulingRepos => <gh.RepositorySlug>{
```

Comment:
If we add more repos, we should consider making this a list of batch scheduling repos. I suspect most repos will want guaranteed scheduling over batch scheduling.